src/docfx/build/link/LinkResolver.cs

```diff
@@ -155,6 +155,10 @@ internal class LinkResolver
                 }
                 return (errors, "", fragment, LinkType.AbsolutePath, null, false);
             }
+            if (resolvedHref.Contains(_config.AlternativeHostName, StringComparison.OrdinalIgnoreCase))
```

Comment:
`Contains` is not safe enough. Is it possible to determine by its hose name?

Reply:
Fixed